test/util/server/server.go

```diff
@@ -430,7 +430,11 @@ func startKubernetesAPIServer(masterConfig *configapi.MasterConfig, clientConfig
 	// we need to set enable-aggregator-routing so that APIServices are resolved from Endpoints
 	kubeAPIServerConfig.APIServerArguments["enable-aggregator-routing"] = kubecontrolplanev1.Arguments{"true"}
 	kubeAPIServerConfig.APIServerArguments["audit-log-format"] = kubecontrolplanev1.Arguments{"json"}
-	go openshift_kube_apiserver.RunOpenShiftKubeAPIServerServer(kubeAPIServerConfig, stopCh)
+	go func() {
+		if err := openshift_kube_apiserver.RunOpenShiftKubeAPIServerServer(kubeAPIServerConfig, stopCh); err != nil {
+			glog.Errorf("openshift-kube-apiserver terminated: %v", err)
```

Comment:
can be a defer to report the not-error termination as well?

Reply:
the non-error termination does not exist. RunOpenShiftKubeAPIServerServer always returns an error, even in the clean case.